backend/config/curation-api.yml

```diff
@@ -323,8 +327,9 @@ paths:
     delete:
       summary: Delete a Dataset or cancel processing for a Dataset in a private Collection or Revision.
       description: >-
-        Delete a Dataset or cancel processing for a Dataset. The **collection_id** MUST reference either a private Collection OR a 
-        Revision. MUST indicate the Dataset by providing the **dataset_id** parameter.
+        Delete or cancel processing for a Dataset with `dataset_id` in `collection_id`. 
+        The **collection_id** MUST reference either a private Collection OR a 
```

Comment:
```suggestion
        The `collection_id` MUST reference either a private Collection OR a 
```

Reply:
I did a quick check - this is the only such instance in the api file of accidentally using asterisks for a variable instead of backticks 👍